includes/Experiments.php

```diff
@@ -365,6 +365,17 @@ public function get_experiments() {
 				'group'       => 'general',
 				'default'     => true,
 			],
+			/**
+			 * Author: @spacedmonkey
+			 * Issue: #6943
+			 * Creation date: 2021-04-07
+			 */
+			[
+				'name'        => 'enablPrePublishVideoOptimization',
```

Comment:
Seems like this is missing an `e`
```suggestion
				'name'        => 'enablePrePublishVideoOptimization',
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7130/commits/f88be1fa76104a9d23c3b9a64201e03f7f859b9b